ElrondNetwork/elrond-go
# Shard scheduled mini blocks pre-validation

* Implemented shard scheduled mini blocks pre-validation
* Added scheduled mini blocks in the proposed body

**How to be tested**

Start a normal system test with txs and sc calls
In epoch 5 scheduled mini blocks mechanism will be activated.
In epoch 6 we should check the log files and messages like: "createScheduledMiniBlocks has been started", "createScheduledMiniBlocks has been finished" and "scheduled mini blocks from me created" should be found
